schema/deploy/insert_todo.sql

```diff
@@ -0,0 +1,14 @@
+-- Deploy todo_app:insert_todo to pg
+-- requires: todos
+-- requires: todo_appschema
+
+BEGIN;
+
+create or replace function todo_app.insert_todo(
```

Comment:
Postgraphile should already expose a mutation that you can call, out of the box:
```graphql
mutation InsertTodo {
  createToDo(
    input: { 
      todo: { task: 'The Task\'s name', completed: false }
    }
  ) 
  {
    # ... whatever you want returned , or '__typename' if nothing
  }
}